integration-tests/docker-compose.yml

```diff
@@ -48,7 +48,7 @@ services:
       - ${TENDERMINT_ZEROFEE_RPC_PORT:-16657}:26657
   chain-abci-zerofee:
     image: "${CHAIN_DOCKER_IMAGE:-integration-tests-chain}"
-    command: "/usr/bin/chain-abci --host=0.0.0.0 --port=26658 --chain_id=${CHAIN_ID} --genesis_app_hash=${ZEROFEE_APP_HASH} --enclave_server=tcp://chain-tx-enclave-zerofee:25933 --data=/.storage"
+    command: "/usr/bin/chain-abci --host=0.0.0.0 --port=26658 --chain_id=${CHAIN_ID} --genesis_app_hash=${ZEROFEE_APP_HASH} --enclave_server=tcp://chain-tx-enclave-zerofee:25933 --tx_query=tcp://chain-tx-enclave-zerofee:25933 --data=/.storage"
```

Comment:
tx-query address != enclave server

just a note to myself, as this will then need to be fixed in https://github.com/crypto-com/chain/pull/637